pkg/console/service_map.go

```diff
@@ -39,10 +39,13 @@ var ServiceMap = map[string]string{
 	"redshift":       "redshiftv2",
 	"sagemaker":      "sagemaker",
 	"ssm":            "systems-manager",
+	"vpc":            "vpc",
+	"dx":             "directconnect/v2"
```

Comment:
Is `dx` a better abbreviation here than `dc`? I don't have much experience with Direct Connect so would be happy to accept your recommendation here @networkprogrammer.

Reply:
Hi @chrnorm , great question. If you were to ask 5 network folks, you are very likely to get 10 answers :)
DX would be the right abbreviation for directconnect.
All directconnect resources start with a dx, and a quick ctrl+f on the below page for directconnect, will show AWS using DX as an acronym

https://aws.amazon.com/directconnect/faqs/